packages/patternfly-4/react-core/src/components/Breadcrumb/BreadcrumbHeading.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Breadcrumb/breadcrumb.css';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import { componentShape } from '../../internal/componentShape';
+
+const propTypes = {
+  /** Content rendered inside the breadcrumb title. */
+  children: PropTypes.node,
+  /** Additional classes added to the breadcrumb item. */
+  className: PropTypes.string,
+  /** HREF for breadcrumb link. */
+  to: PropTypes.string,
+  /** Target for breadcrumb link. */
+  target: PropTypes.string,
+  /** Sets the base component to render. Defaults to <a> */
+  component: componentShape
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  to: null,
+  target: null,
+  component: 'a'
+};
+
+const BreadcrumbHeading = ({ className, children, to, target, component: Component, ...props }) => (
+  <li {...props} className={css(styles.breadcrumbItem, className)}>
+    <h1 className={css(styles.breadcrumbHeading)}>
+      {to && (
+        <Component
+          href={to}
```

Comment:
href and target only make sense for anchor tags, no? So in this case shouldn't the Component always be `a`?

Reply:
Ah, I see @dmiller9911  example above. That's nice then